torch/onnx/_internal/fx/custom_operator.py

```diff
@@ -0,0 +1,58 @@
+from typing import Callable
+
+import torch
+from .function_dispatcher import _ATENLIB_FUNCTIONS, _OP_OVERLOAD_TO_EXPORTER_KEY_TABLE
+
+ONNX_CUSTOM_OP_LIB = torch.library.Library("onnx_custom", "DEF")
+
+
+def _register_onnx_custom_op_overload(schema: str):
+    """This function decorator registers the wrapped function as a `torch._ops.OpOverload` with the provided `schema`.
+
+    Arguments:
+        schema (str): The schema of the wrapped function in PyTorch's type system; e.g., "my_op(Tensor a, Tensor b) -> Tensor c".
+
+    Example:
+    The following example register the `a_funny_op` function as `torch.ops.onnx_custom.my_op` (type: torch._ops.OpOverload).
+
+        import torch
+        import torch.onnx._internal.fx.custom_operator as fx_custom
+        @fx_custom._register_onnx_custom_op_overload("my_op(Tensor a, Tensor b) -> Tensor c")
+        def a_funny_op(a, b):
+            return a + b
+
+        x = torch.randn(3, 4)
+        y = torch.randn(3, 4)
+        # Now, use op_overload to invoke the custom function.
+        z = torch.ops.onnx_custom.my_op(x, y)
```

Comment:
what happens if user still does `z = a_funny_op(a, b)`?

Reply:
Actually looking at https://gist.github.com/ezyang/49fc9c883759573bdb0212236e5b51d9, I think people shouldn't need to explicitly call `torch.ops.onnx_custom.my_op` and it should work when just sticking to `a_funny_op(a, b)`?